R/RLearner_regr_gpfit.R

```diff
@@ -0,0 +1,61 @@
+#' @export
+makeRLearner.regr.gpfit = function(){
+  makeRLearnerRegr(
+    cl = "regr.gpfit",
+    package = "GPfit",
+    par.set = makeParamSet(
+      makeNumericVectorLearnerParam(id = "control", len = 3, lower = c(1, 1, 1)),
+      makeNumericLearnerParam(id = "nug_thres", default = 20, lower = 10, upper = 25),
+      makeLogicalLearnerParam(id = "trace", default = FALSE, tunable = FALSE),
+      makeIntegerLearnerParam(id = "maxit", default = 100, lower = 0),
+      makeUntypedLearnerParam(id = "optim_start", tunable = FALSE),
+      makeLogicalLearnerParam(id = "scale", default = TRUE)
```

Comment:
Soll ich das tunable jetzt doch drin lassen, nachdem Bernd Bischl das
angemerkt hat? Oder rausnehmen? Du bist (in Bezug auf mich) ja der Chef
hier :)




>> @@ -0,0 +1,61 @@
>> +#' @export
>> +makeRLearner.regr.gpfit = function(){
>> +  makeRLearnerRegr(
>> +    cl = "regr.gpfit",
>> +    package = "GPfit",
>> +    par.set = makeParamSet(
>> +      makeNumericVectorLearnerParam(id = "control", len = 3, lower =
>> c(1, 1, 1)),
>> +      makeNumericLearnerParam(id = "nug_thres", default = 20, lower =
>> 10, upper = 25),
>> +      makeLogicalLearnerParam(id = "trace", default = FALSE, tunable =
>> FALSE),
>> +      makeIntegerLearnerParam(id = "maxit", default = 100, lower = 0),
>> +      makeUntypedLearnerParam(id = "optim_start", tunable = FALSE),
>> +      makeLogicalLearnerParam(id = "scale", default = TRUE)
>
> better `makeLogicalLearnerParam(id = "scale", default = TRUE, tunable =
> FALSE)`
>
> ---
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/mlr-org/mlr/pull/916/files/e54b4aca19c2eafc844c62732917d08880da50f8#r65671671